src/mono/mono.proj

```diff
@@ -890,34 +890,69 @@
 
   <Import Project="Directory.Build.targets" />
 
+
   <Target Name="BuildAppleAppBuilder">
     <MSBuild Projects="$(MonoProjectRoot)msbuild\AppleAppBuilder\AppleAppBuilder.csproj"
              Properties="Configuration=$(Configuration)"
              Targets="Restore;Build" />
+    <MSBuild Projects="$(MonoProjectRoot)msbuild\AppleTestRunner\AppleTestRunner.csproj"
+             Properties="Configuration=$(Configuration)"
+             Targets="Restore;Build" />
   </Target>
 
   <UsingTask TaskName="AppleAppBuilderTask" 
              AssemblyFile="$(ArtifactsObjDir)mono\AppleAppBuilder\$(TargetArchitecture)\$(Configuration)\AppleAppBuilder.dll" />
 
-  <Target Name="BuildAppleApp" DependsOnTargets="BuildAppleAppBuilder">
-    <AppleAppBuilderTask 
+  <Target Name="BuildAppleTestBundle" DependsOnTargets="BuildAppleAppBuilder">
+    <PropertyGroup>
+      <TestDir>$(ArtifactsDir)bin\$(TestName)\$(NetCoreAppCurrent)-$(Configuration)</TestDir>
+      <TestDir Condition="!Exists('$(TestDir)')">$(ArtifactsDir)bin\$(TestName)\$(NetCoreAppCurrent)-Unix-$(Configuration)</TestDir>
+      <BclDir>$(ArtifactsDir)bin\runtime\$(NetCoreAppCurrent)-$(TargetOS)-$(Configuration)-$(TargetArchitecture)</BclDir>
+      <BundleDir>$(TestDir)\Bundle</BundleDir>
+    </PropertyGroup>
+    <ItemGroup>
+      <TestBinaries Include="$(TestDir)\*.*"/>
+      <AppleTestRunnerBinaries Include="msbuild\AppleTestRunner\bin\*.*" />
+      <BclBinaries Include="$(BclDir)\*.*" Exclude="$(BclDir)\System.Runtime.WindowsRuntime.dll" />
+      <MonoRuntimeBinaries Include="$(BinDir)\*.*" />
+    </ItemGroup>
+
+    <Error Condition="'$(TestName)' == ''" Text="TestName is not set" />
+    <Error Condition="!Exists('$(TestDir)')" Text="TestDir=$(TestDir) doesn't exist" />
+    <Error Condition="!Exists('$(BclDir)')" Text="BclDir=$(BclDir) doesn't exist" />
+
+    <RemoveDir Directories="$(BundleDir)" />
+
+    <Copy SourceFiles="@(TestBinaries)" DestinationFolder="$(BundleDir)" />
```

Comment:
`SkipUnchangedFiles="true"` for all these? Don't know why that 's not the default.